src/lib/migration/main_workflow.rb

```diff
@@ -120,7 +125,22 @@ def perform_update
     end
 
     def create_snapshot
-      Yast2::FsSnapshot.create_single("before update on migration")
+      out = Yast::SCR.Execute(Yast::Path.new(".target.bash_output"), FIND_CONFIG_CMD)
+
+      log.info("Checking if Snapper is configured: \"#{FIND_CONFIG_CMD}\" returned: #{out}")
+      return :next unless out["exit"] == 0
+
+      cmd = format(CREATE_SNAPSHOT_CMD,
```

Comment:
it is done only at this place, so what is goal to using template with constatns, why not directly have constant?